app/models/prompt.rb

```diff
@@ -99,12 +100,13 @@ def correct_number_of_tags
         # get the tags of this type the user has specified
         taglist = tag_set ? eval("tag_set.#{tag_type}_taglist") : []
         tag_count = taglist.count
+        tag_label = tag_type_label_name(tag_type).downcase
```

Comment:
t ?

Reply:
`ts` is being used on this variable on line 108.